### ISSUE TITLE
FR: select key and edit in a type of val or just merely in string

### ISSUE THREAD

**Issue by witnes0417** [lijyze/obsidian-state-switcher#11]
## further explains: 
- by the **type** I mean like a float or int or just barely string.
- by the **select and edit** I mean compared to the original optional-keyVal way, just select the key and then manually input the value.
- which can improve the liberty

**Comment by latca** [comment]
Just want to add in it would be nice to have the ability to handle boolean values as well.

**Comment by OPEN1004MIND** [comment]
> Just want to add in it would be nice to have the ability to handle boolean values as well.

I agree. Currently, it's all character types, so I'd like to see that improved.
